internal/login/login.go

```diff
@@ -1,18 +1,185 @@
 package login
 
 import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/ecdh"
+	"crypto/rand"
+	"encoding/hex"
+	"encoding/json"
 	"fmt"
+	"io"
+	"net/http"
 	"os"
+	"os/exec"
+	"strconv"
 	"strings"
+	"time"
 
+	"github.com/briandowns/spinner"
+	"github.com/google/uuid"
 	"github.com/spf13/afero"
 	"github.com/supabase/cli/internal/utils"
 	"github.com/supabase/cli/internal/utils/credentials"
 )
 
-func Run(stdin *os.File, fsys afero.Fs) error {
-	accessToken := PromptAccessToken(stdin)
```

Comment:
Are we going to stop prompting for access token completely? This could be a breaking change for users who pipe in tokens from CI environments, for eg. `supabase login <<< "token"`.

This is probably a rare case but I'd prefer keeping backwards compatibility if we can.

Reply:
Good catch, will update.